plugin/src/software/aws/toolkits/eclipse/amazonq/lsp/auth/DefaultLoginService.java

```diff
@@ -167,7 +166,7 @@ CompletableFuture<Void> processLogin(final LoginType loginType, final LoginParam
                 .thenRun(() -> {
                   authStateManager.toLoggedIn(loginType, loginParams, ssoTokenId.get());
                   Activator.getLogger().info("Successfully logged in");
-                  ThreadingUtils.executeAsyncTask(() -> CustomizationUtil.triggerChangeConfigurationNotification());
+                  CustomizationUtil.triggerChangeConfigurationNotification();
```

Comment:
Is this change due to a synchronize issue?

Reply:
It was causing issues in the tests and I realized it is unnecessary since we do want this to happen synchronously to ensure the LSP is using the correct config.